flow/project.py

```diff
@@ -2619,32 +2625,52 @@ def compute_status(data):
                 error_text = str(error)
                 status["completed"] = False
                 status["eligible"] = False
-            result = {
-                "aggregate_id": aggregate_id,
-                "group_name": group.name,
-                "status": status,
-                "_error": error_text,
-            }
+            result = [
+                {
+                    "aggregate_id": aggregate_id,
+                    "group_name": group.name,
+                    "status": status,
+                    "_error": error_text,
+                }
+            ]
+            if (
+                group.name not in self._operations
+                and scheduler_status > JobStatus.unknown
```

Comment:
"unknown" is the first status value in that enum. I think it would be clearer to check `!=`.
```suggestion
                and scheduler_status != JobStatus.unknown
```